CMakeLists.txt

```diff
@@ -50,16 +50,17 @@ if (NOT DEFINED SPIRV_HEADERS_SOURCE_DIR)
 endif()
 
 if (NOT DEFINED SPIRV_TOOLS_SOURCE_DIR)
-
   set(SPIRV_TOOLS_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/third_party/SPIRV-Tools)
-
   use_component(${SPIRV_TOOLS_SOURCE_DIR})
+endif()
 
+if (NOT TARGET spirv-dis)
```

Comment:
This will always be true until we add the SPIRV-Tools subdirectory.  I would just take it away.

Reply:
This won't be true for projects that embed clspv and have already added SPIRV-Tools as clvk does :). You may want to read #202 and #203 for the rationale. Currently supported are the following cases:
- a project embeds clspv and SPIRV-Tools and provides both `SPIRV_TOOLS_SOURCE_DIR` and the target `spirv-dis` since SPIRV-Tools has already been added. Both conditions are false.
- a project embeds clspv and doesn't provide SPIRV-Tools. clspv uses its own version. `SPIRV_TOOLS_SOURCE_DIR` isn't defined so we set the variable to the stock path in third_party/ and the target `spirv-dis` doesn't exist so we add SPIRV-Tools as a sub-directory.

The case that is currently not handled is the one that arises when the user provides `SPIRV_TOOLS_SOURCE_DIR` but wants the build to be managed by clspv's build system. In this case we just want to add SPIRV-Tools using the path provided by the user.

Reply:
> This won't be true for projects that embed clspv and have already added SPIRV-Tools as clvk does :). You may want to read #202 and #203 for the rationale. Currently supported are the following cases:

Ah, quite right.  I had missed that.  Thanks.  In that case, it LGTM.